Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Environment.hs

```diff
@@ -224,7 +225,8 @@ data AppEnv = AppEnv
     ondcTokenHashMap :: HMS.HashMap KeyConfig TokenConfig,
     iosValidateEnpoint :: Text,
     passettoContext :: PassettoContext,
-    quoteRespondCoolDown :: Int
+    quoteRespondCoolDown :: Int,
+    sosAlertsTopicARN :: Text
```

Comment:
Why have we added this on Driver Side, are we using it anywhere on Driver side ?

Reply:
in future if we need this on driver side we can use this